src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Each class created should have a class header comment, refer to Address.java for example.